text/0032-dust-protection/0032-dust-protection.md

```diff
@@ -0,0 +1,126 @@
++ Feature name: dust-protection
++ Start date: 2020-12-07
++ RFC PR: [iotaledger/protocol-rfcs#0032](https://github.com/iotaledger/protocol-rfcs/pull/0032)
+
+
+# Summary
+In the UTXO model, each node in the network needs to keep track of all the currently unspent outputs. When the number of outputs gets too large, this can cause performance and memory issues.
+This RFC proposes a new protocol rule regarding the processing of outputs that transfer a very small amount of IOTA, so-called dust outputs: Dust outputs are only allowed when they are backed up by a certain deposit on the receiving address. This limits the amount of dust outputs, thus making it expensive to proliferate dust. Since a receiver must make a deposit, the protocol makes receiving dust an opt-in feature.
+
+# Motivation
+
+A single typical `SigLockedSingleOutput` as described in [Draft RFC-18](https://github.com/luca-moser/protocol-rfcs/blob/signed-tx-payload/text/0000-transaction-payload/0000-transaction-payload.md) has a size of 42 bytes. When 1 Gi is split up into one billion outputs of 1 IOTA each, this corresponds to about 4 GB of outputs that need to be processed and stored. Since IOTA is feeless, this is far from a prohibitively expensive attack.
```

Comment:
```suggestion
A single typical `SigLockedSingleOutput` as described in [Draft RFC-18](https://github.com/luca-moser/protocol-rfcs/blob/signed-tx-payload/text/0000-transaction-payload/0000-transaction-payload.md) has a size of 42 bytes. When 1 Gi is split up into one billion outputs of 1 IOTA each, this corresponds to about 40 GB of outputs that need to be processed and stored. Since IOTA is feeless, this is far from a prohibitively expensive attack.
```
1 billion * 42 bytes ≈ 40GB

Reply:
yikes, missing 0